src/main/java/org/alfresco/heartbeat/InfoDataCollector.java

```diff
@@ -52,16 +65,34 @@
  *                  <li>minor: String - The minor version number, e.g. 1.<u>2</u>.3. {@link Descriptor#getVersionMinor()}</li>
  *                  <li>patch: String - The version revision number, e.g. 1.2.<u>3</u>. {@link Descriptor#getVersionRevision()}</li>
  *                  <li>hotfix: String - The version label. {@link Descriptor#getVersionLabel()}</li>
+ *                  <li>build: String - The number of the build. {@link Descriptor#getVersionBuild()}</li>
```

Comment:
Not really the number of the build, is more a string that identifies the build.

Reply:
Changes in the next commit.